modules/sagemaker/sagemaker-templates-service-catalog/templates/hf_import_models/seed_code/build_app/ml_pipelines/training/README.md

```diff
@@ -0,0 +1,7 @@
+# Training SageMaker Pipeline
+
+This SageMaker Pipeline definition creates a workflow that will:
+- Prepare the Abalone dataset through a SageMaker Processing Job
+- Train an XGBoost algorithm on the train set
+- Evaluate the performance of the trained XGBoost algorithm on the validation set
+- If the performance reaches a specified threshold, send the model for Manual Approval to SageMaker Model Registry.
```

Comment:
This description isn't right. It applies to the `xgboost_abalone` template, but not this one.

Reply:
Corrected the description. Please review